team-manager/src/test/unit/java/ua/knu/knudev/teammanager/service/DepartmentServiceTest.java

```diff
@@ -151,4 +157,163 @@ void should_throwDepartmentException_When_DepartmentDoesNotExistOnGet() {
 
         verify(departmentRepository, times(1)).findById(TEST_DEPARTMENT_ID);
     }
+
+    @Test
+    @DisplayName("Should create department when given valid data")
+    public void should_createDepartment_When_givenValidData() {
+        SpecialtyCreationDto specialtyCreationDto1 = new SpecialtyCreationDto(14.378, "Computer Engineering");
+        SpecialtyCreationDto specialtyCreationDto2 = new SpecialtyCreationDto(14.378, "Radio technics");
+        SpecialtyCreationDto specialtyCreationDto3 = new SpecialtyCreationDto(14.378, "History");
+
+        DepartmentCreationRequest departmentCreationRequest = buildDepartmentCreationRequest(
+                Set.of(specialtyCreationDto1, specialtyCreationDto2, specialtyCreationDto3),
+                TEST_DEPARTMENT_NAME
+        );
+
+        departmentService.createDepartment(departmentCreationRequest);
+
+        ArgumentCaptor<Department> departmentCaptor = ArgumentCaptor.forClass(Department.class);
+        verify(departmentRepository, times(1)).save(departmentCaptor.capture());
```

Comment:
And put please this verify to the bottom, because our convention is all verifies are at the bottom

Reply:
ok